docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
 
-CONST
-,,,,,
+Syntax
+~~~~~~
+::
+
+  VAR LOGUNIF <min_value> <max_value>
 
-The keyword CONST is used to assign a Dirac distribution to a variable, i.e. set
-it to a constant value. Here is an example of use:
+Parameters
+~~~~~~~~~~
+- **<min_value>**: Specifies the lower bound of the log uniform distribution.
+- **<max_value>**: Sets the upper bound of the log uniform distribution.
 
+Example
+~~~~~~~
+To assign a log uniform distribution ranging from 0.00001 to 1 to a variable:
 ::
 
-   VAR6 CONST 1.0
+   VAR LOGUNIF 0.00001 1
+
+.. image:: fig/loguniform.png
 
-DUNIF
-,,,,,
+Notes
+~~~~~
+The log uniform dstribution is useful when modeling positve variables that are heavily skewed towards a boundary.
 
-The keyword DUNIF is used to assign a discrete uniform distribution. It takes
-three arguments, the number of bins, a minimum and a maximum value. Here is an
-example which creates a discrete uniform distribution with 1, 2, 3, 4 and 5
-as possible values:
+CONST: Dirac Delta Distribution 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The ``CONST`` keyword ensures that a variable always takes a specific, unchanging value.
+
+Syntax
+~~~~~~
 ::
 
-    VAR7 DUNIF 5 1 5
+  VAR CONST <value>
+
+Parameters
+~~~~~~~~~~
+- **<value>**: The fixed value to be assigned to the variable.
+
+Example
+~~~~~~~
+To assign a value of 1.0 to a variable:
+::
+
+   VAR CONST 1.0
+
+.. image:: fig/const.png
+
+DUNIF: Discrete Uniform Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``DUNIF`` keyword assigns a discrete uniform distribution to a variable over a specified range and number of bins.
+
+Syntax
+~~~~~~
+::
 
-Note that you can use the minimum and maximum to scale your distribution. In
-particular this will give you values on the form
+  VAR DUNIF <nbins> <min_value> <max_value>
 
-.. math::
+Parameters
+~~~~~~~~~~
+- **<nbins>**: Number of discrete bins or possible values.
+- **<min_value>**: The minimum value in the range.
+- **<max_value>**: The maximum value in the range.
+
+Example
+~~~~~~~
+To create a discrete uniform distribution with possible values of 1, 2, 3, 4, and 5:
+::
 
-    \textit{min} + i * (max - min) / (nbins - 1)
+   VAR DUNIF 5 1 5
 
-for values of i between 0 and nbins-1.
+.. image:: fig/dunif.png
 
+Notes
+~~~~~
+Values are derived based on the formula:
+:math:`\text{min} + i \times (\text{max} - \text{min}) / (\text{nbins} - 1)`
+Where :math:`i` ranges from 0 to :math:`\text{nbins} - 1`.
 
-ERRF
-,,,,,
+ERRF: Error Function-Based Prior
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ERRF keyword is used to define a prior resulting from applying the error
-function to a normally distributed variable with mean 0 and variance 1. The
-keyword takes four arguments:
+The ``ERRF`` keyword allows creating prior distributions derived from applying the error function to a standard normal variable (with mean 0 and variance 1).
```

Comment:
We do not only apply the error function. I think this is more precise: "...derived from applying the normal CDF (involving the error function) to a standard normal variable. Note that the CDF is not necessarily the standard normal, as `SKEWNESS` and `WIDTH` corresponds to its negative mean and standard deviation respectively. This allows flexibility in creating distributions of diverse shapes and symmetries."

Reply:
Fixed